schedule.html

```diff
@@ -244,7 +244,7 @@ <h4 id="schedule-day-2">
           <tr>
             <td></td>
             <th>
-              08:00–09:00 <span>AM</span>
+              09:00–09:30 <span>AM</span>
```

Comment:
not sure we should start later; I would prefer somewhat longer breaks rather than a delayed start